notebooks/working/6.0-mab-estimate-causal-effect.md

```diff
@@ -0,0 +1,2197 @@
+                                         "Total Travel Distance",
+                                         "Total Travel Cost",
+                                         "Cross Bay Trip",
+                                         "Number of Autos",
+                                         "Number of Licensed Drivers",
+                                         "Household Size",
+                                         "Number of Kids",
+                                         "Utility (Shared Ride 2)"],
+                                  edges=[("Total Travel Distance","Total Travel Time"),
+                                         ("Total Travel Distance","Total Travel Cost"),
+                                         ("Total Travel Time", "Utility (Shared Ride 2)"),
+                                         ("Number of Autos", "Utility (Shared Ride 2)"),
+                                         ("Number of Licensed Drivers", "Utility (Shared Ride 2)"),
+                                         ("Total Travel Cost", "Utility (Shared Ride 2)"),
+                                         ("Household Size", "Utility (Shared Ride 2)"),
+                                         ("Cross Bay Trip", "Utility (Shared Ride 2)"),
+                                         ("Number of Kids", "Utility (Shared Ride 2)")
+    ]
+)
+
+# draw the causal model
+V_Shared_2.draw()
+```
+
+```python
+# Define the causal model
+V_Shared_3p = CausalGraphicalModel(nodes=["Total Travel Time",
+                                         "Total Travel Distance",
+                                         "Total Travel Cost",
+                                         "Cross Bay Trip",
+                                         "Number of Autos",
+                                         "Number of Licensed Drivers",
+                                         "Household Size",
+                                         "Number of Kids",
+                                         "Utility (Shared Ride 3+)"],
+                                  edges=[("Total Travel Distance","Total Travel Time"),
+                                         ("Total Travel Distance","Total Travel Cost"),
+                                         ("Total Travel Time", "Utility (Shared Ride 3+)"),
+                                         ("Number of Autos", "Utility (Shared Ride 3+)"),
+                                         ("Number of Licensed Drivers", "Utility (Shared Ride 3+)"),
+                                         ("Total Travel Cost", "Utility (Shared Ride 3+)"),
+                                         ("Household Size", "Utility (Shared Ride 3+)"),
+                                         ("Cross Bay Trip", "Utility (Shared Ride 3+)"),
+                                         ("Number of Kids", "Utility (Shared Ride 3+)")
+    ]
+)
+
+# draw the causal model
+V_Shared_3p.draw()
+```
+
+```python
+# Define the causal model
+V_WTW = CausalGraphicalModel(nodes=["Total Travel Distance",
```

Comment:
This causal graph is very strange. Shouldn't there be edges from
- travel distance to travel time,
- travel distance to travel cost, and
- travel distance to the utility (wtw)?

Reply:
ah yes, good call. That all makes sense!